data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -404,6 +404,11 @@ run_cluster_bootstrap() {
         --volume /etc/kubernetes:/etc/kubernetes:z \
         "${CLUSTER_BOOTSTRAP_IMAGE}" \
         start --tear-down-early=false --asset-dir=/assets --required-pods="${REQUIRED_PODS}"
+     # remove overrides for installer manifests and restore any user-supplied overrides
+    oc patch clusterversion.config.openshift.io version \
```

Comment:
is it worth putting this in it's own `clear-overrides.done` block, so if the patch fails (e.g. on an API-server hiccup), we don't come back in and re-launch cluster-bootstrap?